src/FluentMigrator.MSBuild/Migrate.cs

```diff
@@ -43,7 +43,7 @@ namespace FluentMigrator.MSBuild
 {
     [CLSCompliant(false)]
     public class Migrate :
-#if NET40 || NET45
+#if NET461
```

Comment:
Reverse the if here, as I think we want AppDomainIsolatedTask as long as the code is compiled in full framework?
I think it will be easier to maintain if we do not have to change the precomiler directives all the times when we change full framework.

Reply:
Good catch. I defined a new constant in Global.proj for .NET Framework (NETFRAMEWORK) to avoid this problems.